src/components/Button.js

```diff
@@ -2,12 +2,29 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = (props) => {
-  const { btname } = props;
-  return <button className="calc" type="button">{btname}</button>;
+  const cssClass = ['calc'];
+  const { btname, wide, color } = props;
+  if (wide) {
+    cssClass.push('basis50');
+  }
+  if (color) {
+    cssClass.push(color);
+  }
+  return (
+    <button className={cssClass.join(' ')} type="button">
+      {btname}
+    </button>
+  );
 };
 
+Button.defaultProps = {
+  color: 'gray',
```

Comment:
The default color should be orange not gray ![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png)